code/modules/mob/living/simple_animal/hostile/skeleton.dm

```diff
@@ -80,4 +80,16 @@
 	maxHealth = 75
 	health = 75
 	color = rgb(114,228,250)
-	loot = list(/obj/effect/decal/remains/human{color = rgb(114,228,250)})
\ No newline at end of file
+	loot = list(/obj/effect/decal/remains/human{color = rgb(114,228,250)})
+
+/mob/living/simple_animal/hostile/skeleton/revenant
+	name = "haunted remains"
+	desc = "It's a skeleton alright, but it seems to be fuelled by haunted energies."
+	gold_core_spawnable = NO_SPAWN
+	maxHealth = 70
+	health = 70
+	var/lastthingtheysaid = "Kill..."
+
+/mob/living/simple_animal/hostile/skeleton/revenant/Life()
```

Comment:
uh. no. just set the skeleton's speak_chance to 10 and add the string to speak(which is a list on simple animals)

Reply:
i see, that makes sense